samples/istio/deployments.yaml

```diff
@@ -0,0 +1,104 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: servicea-deployment
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: servicea
+    spec:
+      containers:
+      - name: servicea
+        image: SERVICE_A_TAG:latest
+        imagePullPolicy: Always
+        ports:
+        - containerPort: 8080
+        envFrom:
+        - configMapRef:
+            name: servicea-config
+        livenessProbe:
+          exec:
+            command:
+            - curl
+            - -f
+            - http://localhost:8080/health
+          initialDelaySeconds: 10
+          periodSeconds: 10
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: servicea-service
+  labels:
+    app: servicea
+spec:
+  ports:
+  - port: 8080
+    name: http
+  selector:
+    app: servicea
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: serviceb-deployment
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: serviceb
+    spec:
+      containers:
+      - name: serviceb
```

Comment:
we need to unify the image name and refer to the docker image created in the sample repo service-mesh-service-b